libflux/src/flux/semantic/flatbuffers/mod.rs

```diff
@@ -1,3 +1,3 @@
-#![allow(clippy::all)]
+#[allow(clippy::all)]
 pub mod semantic_generated;
 pub mod types;
```

Comment:
This file seems to override and remove my current mod.rs file being added to semantic. It created a small conflict the last time I rebased off of your changes. Do you mind removing this change and I'll add it into my PR?

Reply:
As discussed in Slack, I'm going to hold off on merging this until after you put your patch in, just to prevent issues with spurious conflicts. While fixing this lint is important, I'd rather it not interfere or make painful the process of getting actual, important work done. This work should fit in around that work.